src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from "@testing-library/react";
+import SearchResults from "../conponents/SearchResults";
+
+describe("SearchResults", () => {
+  const validProps = {
+    searchResults: ["https://unsplash.com/photos/uIf6H1or1nE", "https://unsplash.com/photos/wCujVcf0JDw"]
+  };
+
+  it("renders correctly", () => {
+    const { asFragment } = render(
+      <SearchResults
+        results={validProps.searchResults} 
+      />
+    );
+    
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("renders the correct number of images", () => {
+    render(
+      <SearchResults
+        results={validProps.searchResults} 
+      />
+    );
+
+    expect(screen.getAllByTestId("search-results")).toHaveLength(2);
```

Comment:
This test is failing because you do not have any `test-id`'s in your code. For this test, I would advise against using a `test-id` as you have other properties you can test for. If you change line 26 to: 
```
expect(screen.getAllByRole("img")).toHaveLength(2);
```
this test will pass